src/SFA.DAS.RoatpOversight.Web/Views/Oversight/AppealOutcome.cshtml

```diff
@@ -1,8 +1,8 @@
 ﻿@using SFA.DAS.RoatpOversight.Domain
-@using SFA.DAS.RoatpOversight.Web.Domain
 @model SFA.DAS.RoatpOversight.Web.Models.AppealViewModel
 @{
     ViewData["Title"] = "RoATP application outcomes - Application summary";
+    bool IsApplicationWasInProgress = !string.IsNullOrEmpty(Model.Appeal.InProgressInternalComments);
```

Comment:
this should be 
var isApplicationInProgress =

I'd prefer it was checking the InProgressDate !=null personally, but not that fussed as I know InProgressInternalComments is also mandatory.

Reply:
I'd echo the same, better to check InProgressDate